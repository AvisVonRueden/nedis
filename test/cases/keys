*3
$3
SET
$3
one
$1
a
-------
*3
$3
SET
$3
two
$1
b
-------
*3
$3
SET
$5
three
$1
c
-------
*2
$4
KEYS
$1
*
