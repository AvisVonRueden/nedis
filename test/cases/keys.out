+OK
+OK
+OK
*3
$3
one
$3
two
$5
three
